README.md

```diff
@@ -6,8 +6,20 @@ Let's see a example from template.json
 
 {
   "name": "b2g-info cmd",
+  "type": "",
```

Comment:
Ya, that's ok. If we give default value, we have to output more default setting value at first beginning of running the boss.